cmd/root.go

```diff
@@ -134,33 +134,42 @@ func namespaceFor(c clientcmd.ClientConfig, overrides *clientcmd.ConfigOverrides
 	return ns, err
 }
 
-// resolveContext returns the server and namespace of the cluster at the provided
-// context. If context is nil, the current context is used.
-func resolveContext(context *string) (server, namespace string, err error) {
+// resolveContext returns the server and namespace of the cluster at the
+// default context.
+func resolveContext() (server, namespace string, err error) {
```

Comment:
This isn't a blocking comment, but you could combine `resolveContext` and `resolveContextFor` since as a user of this internal API, I know what I want, and I may or may not have a context string. This would simplify the internal API and reduce the duplicate a tiny bit.